crates/libs/targets/Cargo.toml

```diff
@@ -24,8 +24,8 @@ windows_i686_gnu = { path = "../../targets/i686_gnu", version = "0.48.0" }
 [target.'cfg(all(target_arch = "x86_64", target_env = "gnu", not(target_abi = "llvm"), not(windows_raw_dylib)))'.dependencies]
 windows_x86_64_gnu = { path = "../../targets/x86_64_gnu", version = "0.48.0" }
 
-[target.'cfg(all(target_arch = "x86_64", target_env = "gnu", target_abi = "llvm", not(windows_raw_dylib)))'.dependencies]
+[target.x86_64-pc-windows-gnullvm.dependencies]
```

Comment:
I'm surprised there's no way to define a target triple with a `cfg` expression. 🤔

Reply:
Yeah, I've searched for that yesterday but also found nothing.